### PR TITLE
Update acp-overview.md

### PR DESCRIPTION
**Pull request by NifemiOmotola**
reworked the ACP Overview page to be more simple and transparent about prerequisites

## PR Description

TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/dg/dev/acp/acp-overview.md [hunk 1]
```diff
@@ -9,13 +9,11 @@ redirect_from:
 keywords: acp
 ---
 
-The App Composition Platform (ACP) lets Spryker Cloud Commerce Operating System (SCCOS) customers connect, configure, and use the available third-party services via apps, in their application with the click of a button, with no or low development efforts from their side.
+The App Composition Platform (ACP) enables Spryker Cloud Commerce Operating System (SCCOS) customers connect, configure, and use the available third-party services available on the platform in their Spryker projects with zero or low development efforts required from their side.
 
-![ACP](https://spryker.s3.eu-central-1.amazonaws.com/docs/aop/app-orchestration-platform-overview/aop.png)
```
Comment (pushokwhite) [inline docs/dg/dev/acp/acp-overview.md:14]: Do we really want to remove this picture?

**Review by pushokwhite — APPROVED** [review]

File: docs/dg/dev/acp/acp-overview.md (hunk 1 above)
Comment (NifemiOmotola) [reply docs/dg/dev/acp/acp-overview.md:14]: Do you know why the picture was added? It's not clear the value it brings also AOP is outdated

Comment (pushokwhite) [reply docs/dg/dev/acp/acp-overview.md:14]: I see that SCCOS can obtain different payment/delivery methods based on what was chosen.
Nevertheless I'm OK to remove it, it looks too heavy and incomprehensible.

**Comment by lenadoc** [comment]
merged in https://github.com/spryker/spryker-docs/pull/2589

**Pull request closed without merging**